<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDashboard</class>
 <widget class="QDialog" name="PatientDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient Dashboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient Information:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Medical Records:</string>
   </property>
  </widget>
  <widget class="QListView" name="MedReclistView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Results of any tests conducted:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>treatment plans:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Appointment Information:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manageAppPshButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>700</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage your appointments</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>700</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Login Page</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_chat">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>550</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>519</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Leave a message</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="messages_view">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>256</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessageButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>660</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Messages Sent</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
